modules/exploits/windows/http/oracle_beehive_prepareaudiotoplay.rb

```diff
@@ -0,0 +1,150 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Oracle BeeHive 2 voice-servlet prepareAudioToPlay() Arbitrary File Upload",
+      'Description'    => %q{
+        This module exploits a vulnerability found in Oracle BeeHive. The prepareAudioToPlay method
+        found in voice-servlet can be abused to write a malicious file onto the target machine, and
+        gain remote arbitrary code execution under the context of SYSTEM. Authentication is not
+        required to exploit this vulnerability.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'mr_me <steventhomasseeley[at]gmail.com>', # Source Incite. Vulnerability discovery, PoC
+          'sinn3r'                                  # MSF module
+        ],
+      'References'     =>
+        [
+          [ 'ZDI', '15-550'],
+          [ 'URL', 'http://www.oracle.com/technetwork/topics/security/cpuoct2015-2367953.html' ]
+        ],
+      'DefaultOptions'  =>
+        {
+          'RPORT'    => 7777
+        },
+      'Platform'       => 'win',
+      'Targets'        =>
+        [
+          ['Oracle Beehive 2', {}]
+        ],
+      'Privileged'     => false,
```

Comment:
From the description, sounds like this should be true, I can fix at merging time